File: Módulo 2 - Arrays/argv.c
#include <stdio.h>
#include <string.h>

int main(int argc, char *argv[])
{
    if (argc == 2)
    {
        // printf("hello, %s!", argv[1]);
        for (int i = 0, n = strlen(argv[1]); i < n; i++)
        {
            printf("%c\n", argv[1][i]);
        }
        printf("hello, %s\n", argv[1]);
    }
    /*
    else
    {
        printf("hello, world");
    }
    */
}

File: Módulo 6 - Python/argv.py
from sys import argv

if len(argv) == 2: # len me informa o comprimento dessa lista (ou seja, do argv)
    print(f"hello, {argv[1]}")
    # python argv.py David - python é o interpretador (não faz parte da execução)
else:
    print("hello, world")
'''
for arg in argv:
    print(arg)
'''
